ethcore/Cargo.toml

```diff
@@ -70,6 +70,9 @@ unexpected = { path = "../util/unexpected" }
 journaldb = { path = "../util/journaldb" }
 tempdir = "0.3"
 
+[dev-dependencies]
+wabt = "0.1"
```

Comment:
I refuse pulling a huge c++ dependency to ethcore as a mandatory dependency. It needs to be do differently.

Reply:
it's not mandatory, it's only for running tests

Reply:
It's mandatory for everyone working on daily basis on the project. It adds `cmake` with all of it's burden to our toolchain.

Reply:
np, but it will be just bytecode dump as a test

we already have c++ build tools prerequisites by the way (rocksdb will be first notable example), they just don't use cmake crate directly and instead use some obscure build scripts, so this cmake reference does not add anything to the list instead of one-time compilation of wabt